### ISSUE TITLE
Weapon_ball etc

### ISSUE THREAD

**Issue by mejmej** [ESX-PUBLIC/esx_outlawalert#1]
Hi, is it possible to get a config option so that we can exclude certain weapons from IsPedShooting ?

**Comment by HumanTree92** [comment]
Yeah i agree with this. Fire Extinguisher is another one. I love it when my Fire Dept goes to put out a Fire & PD is getting reports of shots fired.
